users/users-router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const usersModel = require('../users/users-model');
+const authenticate = require('../middleware/authenticate');
+const jwt = require('jsonwebtoken');
+
+const router = express.Router();
+
+router.get('/', authenticate(), async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
+    const user = jwt.decode(token, { complete: true });
+    const { user_type } = user.payload;
```

Comment:
There may be an error here.  `payload` is the name of the object being encrypted, and it is the value that is returned when the token is decoded.  So `user` is technically `payload`.  I think it should be `const { user_type } = user;`